### PR TITLE
Fix/mz data removal

### LINKED ISSUE #69: Build error due to removed mzData report
Hi, 
http://bioconductor.org/checkResults/3.15/bioc-LATEST/IPO/ shows a build error 
```
Error in .mzRBackendFromContent(x): 
Could not determine file type for .../mtbls2/mzData/MSpos-Ex1-Col0-48h-Ag-1_1-A,1_01_9818.mzData
```
This is because mzData was droppen in mzR/xcms in mzR-2.29.3
Fix: Move to mzML instead. 
Yours, Steffen

### PR DESCRIPTION
**Pull request by sneumann**
Hi, move from mzData to mzML. Closes #69

### CODE REVIEW

File: R/zzz.R [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.onAttach <- function(libname, pkgname) {
+    msg <- sprintf(
+        "Package '%s' is deprecated and will be removed from Bioconductor
```
Comment (rietho) [inline R/zzz.R:3]: Have you @sneumann  checked with Christoph on that?

He reached out recently to ask if I could update IPO, but I do not have time for that. If the idea is to continue to offer IPO, we'd need to find a new place for it.

**Comment by rietho** [comment]
Thanks @sneumann. Can you confirm that this code change should work? I am not performing any checks on the code. But if you give your go, I'll merge the code.

**Comment by rietho** [comment]
Thanks. I just merged the PR.

I'll check back after my upcoming vacation to update the maintainer email.

**Comment by lshep** [comment]
@rietho  Don't undeprecate yourself --  please make sure it is building without ERROR on the Bioconductor RELEASE_3_15 branch and then request undeprecation at bioc-devel@r-project.org  cc: lori.shepherd@roswellpark.org  if you plan to keep it in Bioconductor.  It has been removed from devel 3.16 and will only be added back after undeprecation on the RELEAES_3_15 branch

**Comment by rietho** [comment]
Thanks @lshep for the clarification on the un-deprecation process.

**Pull request merged**